actors/builtin/power/power_test.go

```diff
@@ -441,15 +441,16 @@ func TestPowerAndPledgeAccounting(t *testing.T) {
 
 		power64Unit, err := builtin.ConsensusMinerMinPower(abi.RegisteredPoStProof_StackedDrgWindow64GiBV1)
 		require.NoError(t, err)
-		assert.True(t, power64Unit.GreaterThan(powerUnit))
+
```

Comment:
Remove this subtest which is now testing deprecated behavior ("consensus power depends on proof type")